components/dashboard/DashboardSection.tsx

```diff
@@ -84,6 +86,9 @@ const DashboardSection = ({ account, isLoading, section, subpath }) => {
     if (section === SECTIONS.TRANSACTIONS && useNewTransactionsPage) {
       DashboardComponent = AccountTransactions;
     }
+    if (section === SECTIONS.OVERVIEW && isIndividualAccount(account)) {
```

Comment:
If SECTIONS is no longer a 1:1 map, we should consider turning SECTIONS into a function that resolves the correct component.

Reply:
Yeah, that'd make sense! Will save this for a follow up